testcases/pkcs11/getobjectsize.c

```diff
@@ -93,7 +95,7 @@ CK_RV do_GetObjectSize(void)
         return rc;
     }
 
-    printf("C_GetObjectSize test passed\n");
+    testcase_pass("C_GetObjectSize test passed\n");
```

Comment:
just saw that I can remove the newline in testcase_pass, there are enough newline in the macro ... will check all places.